_data/projects/Nuclear.yml

```diff
@@ -0,0 +1,16 @@
+name: Nuclear
+desc: Desktop music player for streaming from free sources
+site: https://nuclear.js.org
+tags:
+- JavaScript
+- js
+- web
+- React
+- Electron
+- music
+- Linux
+- Mac
+- Windows
+upforgrabs:
+  name: good first issue
+  link: https://github.com/nukeop/nuclear/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
@nukeop Please change this to https://github.com/nukeop/nuclear/labels/good%20first%20issue

Reply:
Done